src/pytest_html/html_report.py

```diff
@@ -0,0 +1,342 @@
+import bisect
+import datetime
+import json
+import os
+import re
+import time
+from collections import defaultdict
+from collections import OrderedDict
+
+from py.xml import html
```

Comment:
can we get rid of that somehow, pylib is "dead"

Reply:
Hi @RonnyPfannschmidt 

Yes, the point of "Next Gen" is to get rid of that, and other things. So basically, this whole file will be replaced, if not in its entirety, then a lot of it.

I appreciate you taking a look, but this is far from ready. I should've set it to "Draft". Apologies 🙏

Reply:
i was curious anyway ^^ i'd love to have a little talk/chat about this sometime soon

Reply:
Let me get it to a workable state, and I'm happy to chat @RonnyPfannschmidt! I know @davehunt wants to join as well.